static/app/views/starfish/views/spans/selectors/actionSelector.tsx

```diff
@@ -83,34 +83,21 @@ const LABEL_FOR_MODULE_NAME: {[key in ModuleName]: ReactNode} = {
 };
 
 function getEventView(location: Location, moduleName: ModuleName, spanCategory?: string) {
-  const queryConditions: string[] = [];
-  queryConditions.push(`has:${SPAN_DESCRIPTION}`);
-
-  if (moduleName) {
-    queryConditions.push(`has:${SPAN_ACTION}`);
-  }
-
-  if (![ModuleName.ALL, ModuleName.NONE].includes(moduleName)) {
-    queryConditions.push(`${SPAN_MODULE}:${moduleName}`);
-  }
-
-  if (moduleName === ModuleName.DB) {
-    queryConditions.push(`!${SPAN_OP}:db.redis`);
-  }
-
-  if (spanCategory) {
-    if (spanCategory === NULL_SPAN_CATEGORY) {
-      queryConditions.push(`!has:span.category`);
-    } else if (spanCategory !== 'Other') {
-      queryConditions.push(`span.category:${spanCategory}`);
-    }
-  }
+  const query = buildEventViewQuery(
+    moduleName,
+    location,
+    undefined,
+    undefined,
+    spanCategory
+  )
+    .filter(Boolean)
```

Comment:
Is filtering by Boolean necessary here? I would find it a bit odd that `buildEventViewQuery` might return empty conditions and that isn't handled in `buildEventViewQuery` itself.

Reply:
Not necessary here, updated!